### PR TITLE
fix: qb_8718_9133 added x and y axis instructions

### PR DESCRIPTION
**Pull request by iqi-qlik**
[QB-8718](https://jira.qlikdev.com/browse/QB-8718) 
[QB-9133](https://jira.qlikdev.com/browse/QB-9133) 

Screen reader instructions for X and Y-axis dropdown

## Description
In order to understand how to access the x and y-axis properties, the screen reader should announce the current menu option as well as an instruction message that tells the user how to navigate.

## Fix
Added `aria-label` 
Tested on NVDA screen reader.

[Bug fix Video-1](https://jira.qlikdev.com/secure/attachment/280563/qb_8718_9133_fix_blank_reading.mp4)

[Bug fix Video-2](https://jira.qlikdev.com/secure/attachment/280569/qb_8718_9133_fix_blank_reading_ref2.mp4)

Screenshot 2022-05-17 at 11 11 28 AM

### CODE REVIEW

**Comment by afridell** [comment]
You need to add unit tests

**Comment by afridell** [comment]
you need to add unit tests to pass the codeclimate check

**Comment by iqi-qlik** [comment]
> you need to add unit tests to pass the codeclimate check

okay

**Comment by tasseKATT** [comment]
See comments here: https://github.com/qlik-trial/la-vie/pull/4289

**Comment by jjxie** [comment]
Check comments from https://github.com/qlik-trial/la-vie/pull/4334.

File: src/picasso-components/react-components/data-title-component.jsx [hunk 1]
```diff
@@ -92,7 +92,7 @@ export default function createDataTitileComp() {
           style: { minWidth: '250px', maxHeight: '300px' },
         }}
       >
-        <List dense dir={dir} component="nav">
+        <List dense dir={dir} aria-label="use the tab keys to navigate between the options" component="nav">
```
Comment (jjxie) [inline src/picasso-components/react-components/data-title-component.jsx:95]: This instruction should be translated

Comment (jjxie) [reply src/picasso-components/react-components/data-title-component.jsx:95]: After you translated it, translate same place in la-vie

Comment (iqi-qlik) [reply src/picasso-components/react-components/data-title-component.jsx:95]: ok

File: src/picasso-components/react-components/data-title-component.jsx [hunk 2]
```diff
@@ -244,9 +241,12 @@ export default function createDataTitileComp() {
       if (minWidth) {
         style.minWidth = minWidth;
       }
+
```
Comment (jjxie) [inline src/picasso-components/react-components/data-title-component.jsx:244]: Could you remove unnecessary new line changes? Line 246 too

Comment (iqi-qlik) [reply src/picasso-components/react-components/data-title-component.jsx]: ok. Removed.

File: src/picasso-components/react-components/data-title-component.jsx [hunk 3]
```diff
@@ -245,8 +242,10 @@ export default function createDataTitileComp() {
         style.minWidth = minWidth;
       }
       const dir = rtlUtils.detectTextDirection(titleData.text);
+
```
Comment (jjxie) [inline src/picasso-components/react-components/data-title-component.jsx:245]: this line better removed too

Comment (iqi-qlik) [reply src/picasso-components/react-components/data-title-component.jsx:245]: ok

**Review by jjxie — APPROVED** [review]

**Pull request merged**